src/citation_vs_reviewscore.py

```diff
@@ -0,0 +1,15 @@
+import csv
+import os
+import sys
+
+src_path = os.getcwd()
```

Comment:
it is better to use Path() in pathlib for  these things.  Good habit to get into to start using that now.

Reply:
I have looked into the pathlib documentation and agree with you. Berrak and I will implement this.